examples/widgets/indexed_stack.dart

```diff
@@ -0,0 +1,77 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:sky/material.dart';
+import 'package:sky/rendering.dart';
+import 'package:sky/widgets.dart';
+
+class IndexedStackDemo extends StatefulComponent {
+  IndexedStackDemo({ this.navigator });
+  final NavigatorState navigator;
```

Comment:
For consistency with other StatefulComponent classes, put a newline after the constructor and one before createState().